### ISSUE TITLE
Suggestion: Tutorial MD should be part of this project

### ISSUE THREAD

**Issue by colin-riddell** [solid/profile-viewer-tutorial#3]
Hi,

I think the tutorial here: https://solid.inrupt.com/docs/app-on-your-lunch-break - should have source put in this repo'. I'd like to be able to make PR's on it.

**Comment by RubenVerborgh** [comment]
We should probably link from this repo to that tutorial. Feel free to add to README.

**Comment by nealmcb** [comment]
Yes, I'd also like to be able to comment on the tutorial instructions themselves, which aren't here.

In particular, I was very puzzled when following the different steps. From the names like `tutorials/lunch-break/steps/01` tt seemed clear that they were pointing to path names within the repo, which I couldn't find for the life of me.  Finally I saw they were referring not to files but to tags.

That's a very clever way to make it easy to follow, but only when I realize how to easily use it. Clicking on a tag in a github site isn't easy to use, since it leads us to do copy-paste.

But telling us to clone the repo, and then, e.g.

    git checkout tutorials/lunch-break/steps/08

and refresh the browser is an easy approach.

I've added that to the README.md which some people will notice.  Referring to it from the tutorial itself would be better.

Thanks!

**Comment by RubenVerborgh** [comment]
It depends no how people want to follow it. My idea would be rather that they try to write the code themselves, right?

**Comment by nealmcb** [comment]
Often, probably so. I was curious to start by just observing the various pieces going together, and getting a sense from that what it was all about at first, rather than writing code in a language which is not my favorite.
At any rate my main goal was to be clearer about the mechanics. Effective pedagogy for tutorials is something I'll leave for others.
Thanks!

**Comment by aaaagile** [comment]
Wouldike to ftp to my solid pod to Control code.. 
How would I achieve that?

**Comment by RubenVerborgh** [comment]
> Wouldike to ftp to my solid pod to Control code..

That is currently not supported.

**Comment by aaaagile** [comment]
OK, Thanks for the quick response 😊

**Comment by aaaagile** [comment]
So
I would expect this to happen in the fullness of time?
Big fan of tbl!
Would like to revamp my website as I have spent my working life, before
retirement, expounding his work at cern.
djb.mypi.co
Its my home site on a raspberry pi
So you will see that I want to 'inrupt' it. 😊
Regards
DJB
